source/_integrations/lamarzocco.markdown

```diff
@@ -107,3 +116,34 @@ Host:
 |-------------|-------------| ------------------------| ---------------------- |
 | Prebrew/-infusion mode | Whether to use prebrew, preinfusion, or neither | Disabled, Prebrew, Preinfusion | Linea Micra, Linea Mini, GS3 AV |
 | Steam level | The level your steam boiler should run at | 1,2,3 | Linea Micra |
+
+## Supported devices
+Currently only devices from the `Home` range are supported. 
+
+That means
+
+- Linea Mini
+- Linea Mini R
+- Linea Micra
+- GS3 AV
+- GS3 MP
+
+## Possible use-cases
+- Control your machine through voice, allowing you to change boiler temperatures quickly without opening the app.
+- Control your smart coffee scales (tare/timer start) when a brew starts.
+- Turn on lights next to the machine while a brew is running.
+  
+## Known Limitations
```

Comment:
```suggestion
## Known Limitations

```

Reply:
I don't see the difference (neither doe GitHub because I can't commit the change)

Reply:
Ah newline